src/module_1/module_1_meteo_api.py

```diff
@@ -100,7 +95,7 @@ def daily_to_yearly(daily):
     return yearly
 
 
-def plotting(list):
+def plotting(list, city):
```

Comment:
Type hinting and docstrings

Reply:
This method is entangling some postprocessing and plotting. It is good practices to split this two concepts into different methods:
 - 1 Method computes the final data to plot
 - 2 method does all of the plot and potentially some *minor* tweaks if necessary to the data